packages/bulk-edit/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@theia/bulk-edit",
+  "version": "1.5.0",
```

Comment:
The version should be updated to match the rest of the application, namely `1.6.0` at this time.

Reply:
Fixed. Thanks